channels/fr.m3u

```diff
@@ -291,8 +291,12 @@ http://51.210.199.59/hls/stream.m3u8
 http://51.210.199.16/hls/stream.m3u8
 #EXTINF:-1 tvg-id="PokerTV.fr" tvg-country="IR" tvg-language="" tvg-logo="https://i.imgur.com/l5fZQ3V.png" group-title="Sports",Poker TV [Not 24/7]
 http://51.210.199.17/hls/stream.m3u8
+#EXTINF:-1 tvg-id="PressTV.fr" tvg-country="FR" tvg-language="French" tvg-logo="https://i.imgur.com/5tAWpM3.png" group-title="News",Press TV (1080p)
+https://live1.presstv.ir/live/presstvfr/index.m3u8
 #EXTINF:-1 tvg-id="RMCDecouverte.fr" tvg-country="FR" tvg-language="French" tvg-logo="https://i.imgur.com/2pZBr1Y.jpg" group-title="",RMC Découverte (720p) [Offline]
 https://s6.tntendirect.com/rmcdecouverte/live/playlist.m3u8
+#EXTINF:-1 tvg-id="RTFrance.fr" tvg-country="FR" tvg-language="French" tvg-logo="https://i.imgur.com/WcUS9t8.png" group-title="News",RT France (1080p)
```

Comment:
Already in `channels/ru.m3u`
https://github.com/iptv-org/iptv/blob/df7907497b48d3fb7d91d566d3855136ca88e1ea/channels/ru.m3u#L121-L122